microprofile/cdi/src/main/resources/META-INF/helidon/module.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "CDI",
+    "flavor": "MP",
+    "feature": true,
```

Comment:
So the presence of a `module.json` file does not itself denote a feature?

Reply:
oops, this should not have been commited. The presence on classpath is not sufficient - the feature must be registered. That is why features are added using a static initializer. This is intended for started and CLI. I will remove it from the commit.